priv/apps/itest/lib/in_flight_exit_client.ex

```diff
@@ -324,7 +325,9 @@ defmodule Itest.InFlightExitClient do
     }
 
     {:ok, receipt_hash} = Ethereumex.HttpClient.eth_send_transaction(txmap)
-    wait_on_receipt_confirmed(receipt_hash, @retry_count)
+    wait_on_receipt_confirmed(receipt_hash)
+
+    get_next_exit_from_queue()
```

Comment:
Need `get_next_exit_from_queue()` twice?

Reply:
actually, I can delete the whole file!